packages/plugin-avatar/README.md

```diff
@@ -0,0 +1,151 @@
+# AvatarUrlBatcher
```

Comment:
we're no longer generating READMEs. Please delete this content and put a small summary in (see e.g. the convo plugin for an example).